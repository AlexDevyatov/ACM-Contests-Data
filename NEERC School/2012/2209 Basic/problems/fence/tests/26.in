10
-69 -63
-41 -67
3 -70
67 -73
74 -72
75 9
72 44
65 54
-24 73
-69 65
-30 -58
