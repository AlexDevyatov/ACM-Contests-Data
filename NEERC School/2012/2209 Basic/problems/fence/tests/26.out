11
-69 -63
-41 -67
3 -70
67 -73
-30 -58
74 -72
75 9
72 44
65 54
-24 73
-69 65
